bird.cpp

```diff
@@ -39,10 +39,14 @@ void Bird::draw()
 {
 	glDepthMask(GL_FALSE);
 
-	glm::mat4 identity = glm::mat4(1.0f);
-	glm::mat4 result = identity;
+	// Pitch (forward tilt in planespeak) calculation.
+	float pitch = (float) atan(velocity.y)*5000; // TODO Trial-and-error value
```

Comment:
Why not use velocity directly? Something like `float angle = acos(glm::fastNormalizeDot(velocity, glm::vec3(1.f, 0.f, 0.f)))` should give you the angle directly.